spec-main/chromium-spec.ts

```diff
@@ -561,12 +561,11 @@ describe('chromium features', () => {
           w.show();
         }
 
-        w.webContents.once('new-window', (e, url, frameName, disposition, options) => {
-          expect(options.show).to.equal(w.isVisible());
-          w.close();
-          done();
-        });
+        const newWindow = emittedOnce(w.webContents, 'new-window');
         w.loadFile(path.join(fixturesPath, 'pages', 'window-open.html'));
+        const [,,,, options] = await newWindow;
+        expect(options.show).to.equal(w.isVisible());
+        w.close();
```

Comment:
Won't be closed in the expectation fails.

Reply:
https://github.com/electron/electron/pull/24019 fyi